examples/news-list/common/sagas.js

```diff
@@ -0,0 +1,28 @@
+function mainSaga({ executeCommand }) {
```

Comment:
Move sagas to separated package (add issue)

Reply:
It's just end-programmer defined saga function entry point. Like there are aggregates, read-models, view-models, and it's declaration for server-side sagas.
Which package is meant for extraction? Like `resolve-saga`?

Reply:
Yes, i mean `resolve-saga` package

Reply:
(Extracted to card)